lizard_auth_client/templates/lizard_auth_client/management/base.html

```diff
@@ -0,0 +1,82 @@
+{% load i18n %}
+{% get_current_language as LANGUAGE_CODE %}
+{% load static %}
+
+<!doctype html>
+<html lang="{{ LANGUAGE_CODE }}">
+  <head>
+    <meta charset=utf-8>
+    <title>{% block title %}Management{% endblock %}</title>
+
+    <!-- jQuery Core 3.1.1 minified -->
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js"
+    crossorigin="anonymous"></script>
+
+    <!-- Bootstrap compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
```

Comment:
Nou breekt m'n klomp. Ik heb zelf een `base.html` toegevoegd die je als site kan customizen door zelf je eigen js/css toe te voegen. Want we wilden absuluut geen basis js/css meer (à la lizard-ui), alles moest met javascript.

En nou zit hier een complete js/css head in met een specifieke versie van bootstrap. En crispy forms.

En een `main.css`.

Wordt ik nou gek?